include/gridtools/fn/column_stage.hpp

```diff
@@ -101,5 +101,13 @@ namespace gridtools::fn {
     using column_stage_impl_::column_stage;
     using column_stage_impl_::fwd;
     using column_stage_impl_::merged_column_stage;
+#if defined(__NVCC__) && (__CUDACC_VER_MAJOR__ == 12 && __CUDACC_VER_MINOR__ >= 1 && __CUDACC_VER_MINOR__ <= 2)
+    // workaround CTAD issue in CUDA 12.1, 12.2 (https://github.com/GridTools/gridtools/issues/1766)
```

Comment:
The comment is good, but if you declare the workaround condition as a "variable" then no need for it.